app/models/web_hook_event_type.rb

```diff
@@ -1,21 +1,77 @@
 # frozen_string_literal: true
 
 class WebHookEventType < ActiveRecord::Base
-  TOPIC = 1
```

Comment:
This scares me a little...even with the migration. I think maybe it would be safer to leave these in this PR, then make a followup PR to delete them.

Reply:
But of course still make your new `TYPES` const at the same time, I just mean leave these old ones in for now.